### PR TITLE
refactor: add account param to methods

### PR DESCRIPTION
**Pull request by fbwoolf**
Proposing to add `account` here to method params, and updating/changing the PSBT params.

cc @kyranjamie

### CODE REVIEW

File: packages/types/src/methods/send-transfer.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { DefineRpcMethod, RpcRequest, RpcResponse } from '../rpc';
 
 export interface SendTransferResponseParams {
```
Comment (fbwoolf) [inline packages/types/src/methods/send-transfer.ts:3]: Noticed here, shouldn't the naming be `SendTransferRequestParams` (and same for others that show `Response`?

File: packages/types/src/methods/sign-psbt.ts [hunk 2]
```diff
@@ -3,11 +3,9 @@ import { DefineRpcMethod, RpcRequest, RpcResponse } from '../rpc';
 export type SignPsbtRequest = RpcRequest<'signPstb'>;
```
Comment (kyranjamie) [inline packages/types/src/methods/sign-psbt.ts:3]: Another typo here fyi

File: packages/types/src/methods/sign-psbt.ts [hunk 3]
```diff
@@ -3,11 +3,9 @@ import { DefineRpcMethod, RpcRequest, RpcResponse } from '../rpc';
...
 
 export type SignPsbtResponse = RpcResponse<{
-  psbtHex: string;
-  inputToSign: {
-    sigHash: any[];
-  }[];
-  network: any;
+  account: number;
+  hex: string;
+  signAtIndex: number | number[];
```
Comment (kyranjamie) [inline packages/types/src/methods/sign-psbt.ts:8]: optional type,

what about sighash and public key?

File: packages/types/src/methods/send-transfer.ts (hunk 1 above)
Comment (kyranjamie) [reply packages/types/src/methods/send-transfer.ts:3]: yes it should

**Review by kyranjamie — APPROVED** [review]
Thanks for the PR @fbwoolf

**Comment by janniks** [comment]
Is `account` or `address` better? (Or both)

File: packages/types/src/methods/sign-psbt.ts (hunk 3 above)
Comment (fbwoolf) [reply packages/types/src/methods/sign-psbt.ts:8]: Forgot `allowedSighash`, thanks. I think `publicKey` came from the Connect implementation of PSBT signing. I was confused in our code why it was a param? The app didn't explicitly pass it to us? I don't see that it is ever used during signing?

**Comment by fbwoolf** [comment]
> Is `account` or `address` better? (Or both)

@kyranjamie can you comment on this as a pref? In our code we used the `accountIndex` not the address.

File: packages/types/src/methods/sign-psbt.ts (hunk 3 above)
Comment (fbwoolf) [reply packages/types/src/methods/sign-psbt.ts]: I am also questioning `network` in our implementation, where is it ever used?

**Comment by fbwoolf** [comment]
Also, proposing here to add the `opReturn` string param to `sendTransfer`.

**Review by kyranjamie — APPROVED** [review]

**Comment by kyranjamie** [comment]
Not sure on address vs accountIndex, no real pref. Is one easier than the other?

**Comment by fbwoolf** [comment]
Can this get merged and released?

**Pull request merged**